addons/binding/org.openhab.binding.mihome/ESH-INF/thing/bridge.xml

```diff
@@ -16,22 +18,25 @@
         <config-description>
```

Comment:
@pboos I'd still opt for mergin togeter `bridge` and `gateway`. 
Take a look e.g. at [Z-Wave Serial Controller](https://github.com/openhab/org.openhab.binding.zwave/blob/3133e344fea4299720f06d0bf26181bdb177496c/ESH-INF/thing/_controller_serial.xml) - it's a matter of passing `channel-type` configurations to the bridge.xml file.

I'm not sure how to refactor `XiaomiBridgeHandler` to match the logic though.